### DISCUSSION TITLE
How does one use RoyalRoad properly?

### DISCUSSION THREAD

**Discussion by GreatestFool** [lncrawl/lightnovel-crawler#2136] [Q&A]
I wanted to archive this [book series](https://www.royalroad.com/fiction/49879/paranoid-mage) since I've finished reading it, and I found this project recently. I can download everything by simply running `lncrawl -s https://www.royalroad.com/fiction/49879/paranoid-mage`, but the output is not exactly fitting given how the series is structured on RoyalRoad. It does not seem to be able to grab the volumes properly, so epub generation ends up adding chapters from 1 to 100, and 101 to 110, instead of individual files for the various volumes. I've also tried using `--multi`, but no luck so far.

Is there just something I'm missing, or will I need to manually specify ranges and download that way?

**Comment by tward1505** [comment]
As far as I can tell the program's volumes are basically every 100 chapters, so if you want a specific volume according to the book, yes a specific chapter range for that volume, combined into 1 file, then rinse and repeat for following volumes.

for provided example I would say use the option "select specific chapters", as they are named easily enough to figure out when a volume ends and what isn't actually a chapter. Then just run it using the toc url you linked to find it in the program  over and over for each volume.
